install/kubernetes/cilium/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 1.9.90
-appVersion: 1.9.90
+version: 1.10.90
```

Comment:
shouldn't this be `1.11.90` for master?

Reply:
No, it's 1.10.90 as we haven't released 1.11 yet. See it as the `90` is between `1.10.x` and `1.11.0`